### ISSUE TITLE
apply different methods on same dataset

### ISSUE THREAD

**Issue by kjgonzalez** [kjgonzalez/codefiles#5]
mess with iris dataset. use different algorithms to identify the flower types. 

- [x] pre-developed NN
- [x] bayesian network
- [x] random forest

* 150 total samples, 30 for testing
* NN achieved about 0.933 accuracy
* G-NB achieved about 0.933 accuracy
* RF achieved about 0.933 accuracy

final lessons learned: 
* RF is very flexible, but takes a LOT of effort to program correctly. 
* with enough tuning, networks can achieve similar performance, but are optimized for different tasks
* with high variance comes overfitting to training data, per Elements of Statistical Learning. fix by ending training early or adding more bootstrapped trees to average.
* naive bayes classifier is nice and simple to program, with very few hyperparameters, but makes several assumptions (esp independent variables)

input: training and testing data
output:
--- "performance: 0.XXX"

**Comment by kjgonzalez** [comment]
all three networks built by hand and tested.
